policyeval/commands.go

```diff
@@ -39,6 +43,30 @@ func (pe *PolicyEvaluator) HandleCommand(ctx context.Context, evt *event.Event)
 			}
 		}
 		pe.sendSuccessReaction(ctx, evt.ID)
+	case "!leave":
```

Comment:
This accidentally made its way in from an unstaged commit from #1, needs removing